lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystem.cs

```diff
@@ -45,9 +45,11 @@ public void TestCreateNotSupported()
         }
 
         [Fact]
-        public void TestOpenNotSupported()
+        public void TestOpen()
         {
-            Assert.Throws<NotSupportedException>(() => new TestContext().GetAzureFileSystem().Open(FakeUri));
+            var testContext = new TestContext();
+            testContext.GetAzureFileSystem().Open(new Uri(FakeUri, "container/file"));
+            testContext.TestCloudBlockBlob.Received(1).Open();
```

Comment:
Make sure to add assert logic to the Open result.

Reply:
Nothing to Assert on the Open result as GetAzureFileSystem() returns a proxy which return null on the Open method call